src/main/scala/ai/privado/languageEngine/java/tagger/sink/JavaAPITagger.scala

```diff
@@ -138,8 +143,18 @@ class JavaAPITagger(cpg: Cpg) extends PrivadoSimplePass(cpg) {
   }
 
   private def httpPackagesInImport(): Boolean = {
-    val imports = ImportUtility.getAllImportsFromProject(AppCache.localScanPath, Language.JAVA)
-    imports.size match {
+    val imports          = ImportUtility.getAllImportsFromProject(AppCache.localScanPath, Language.JAVA)
+    val httpPackageRegex = commonHttpPackages.r
+    val httpPackages     = mutable.HashSet[String]()
+
+    imports.par.foreach((statement) => {
+      httpPackageRegex.findFirstIn(statement) match {
+        case Some(value) => httpPackages.addOne(value)
+        case _           => ()
+      }
+    })
+
+    httpPackages.size match {
```

Comment:
If HTTP package size is nonZero, it means Http packages are present, the current implementation is opposite